app/src/main/res/color/material_button_background.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The name of this file seems very generic. It doesn't give much info about when to use it. Maybe adding primary or blue or something that can help us knowing when to use this background?

Reply:
Changing it to be broken site specific.